content/en/user-guide/aws/kinesis/index.md

```diff
@@ -1,57 +1,194 @@
 ---
 title: "Kinesis"
-categories: ["LocalStack Community"]
+linkTitle: "Kinesis"
 description: >
-  Get started with AWS Kinesis on LocalStack
-aliases:
```

Comment:
Removing the alias means that old links might break, doesn't it?

Reply:
We are safely removing the aliases, since Google and other search engines have indexed the new URLs.